pkg/version/version.go

```diff
@@ -23,3 +32,92 @@ func MaybePrintVersionAndExit(printVersion bool) {
 	}
 	log.Infof("running conduit version %s", Version)
 }
+
+type versionStatusChecker struct {
+	version string
+	client  pb.ApiClient
+}
+
+func (v versionStatusChecker) SelfCheck() (checks []*healthcheckPb.CheckResult) {
+	cliVersion := v.version
+	cliIsUpToDate := &healthcheckPb.CheckResult{
+		Status:           healthcheckPb.CheckStatus_OK,
+		SubsystemName:    "conduit-version",
+		CheckDescription: "CLI is up-to-date",
+	}
+
+	latestVersion, err := getLatestVersion()
```

Comment:
I find the name of this variable a little confusing, as it clashes with the `latestVersion ` struct. Maybe call it something else?

Reply:
Have left this the same given the `latestVersion` struct is no more.